README.md

```diff
@@ -68,6 +68,11 @@ __Important__ - _all_ AEP SDK Android binaries depend on `core-1.x.x.aar`.
 
 ## Installing the AEP SDK for iOS
 
+> __Important__ 
+> 
+> ⚠️ A newer version of the [AEP Mobile SDKs (3.x)](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift) is now available! These SDKs are open source, bring many exciting new features and include support for Swift Package Manager (SPM), CocoaPods and XCFrameworks.
+> If you are currently using our Objective-C (ACP-prefix) libraries, please see the [Migrating to Swift](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift) guide for next steps.
+
 > __Important__ 
```

Comment:
Can these two "Important" messages get merged into one?
Also, can the first sentence of the second important message be removed or clarified?  It refers only to the AEP SDK, but the ACP SDK is also distributed using XCFrameworks.

Reply:
This entire page refers to AEP SDKs, did some renaming to indicate the instructions are mainly for 2.x